src/droplet.F

```diff
@@ -545,20 +551,62 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
      !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
 
      allocate(holes_ind(numHoles))
+!$acc enter data create(holes_ind)
 
-     i=0
 
+     !$acc parallel loop gang vector default(present) 
+     do np=1,nparcelsLocal
+       msk(np) = (pdata_locind(np,1) .eq. undefined_index) .and. &
+                 (pdata_locind(np,2) .eq. undefined_index)
+     enddo
```

Comment:
For consistency, add `!$acc end parallel`.

Reply:
@sjsprecious Actually there is no consistency throughout CM1 regarding the use of '$acc end parallel'.  I prefer to not include it so as to minimize the actual number of $acc directives present in the code.  I don't think it adds anything with respect to clarity.

Reply:
I see. But I still think `!$acc end parallel` is helpful for clarity, especially when we have multiple kernels consecutively or have a very long kernel.